hbase-common/src/main/java/org/apache/hadoop/hbase/util/ClassSize.java

```diff
@@ -29,7 +29,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.apache.hbase.thirdparty.com.google.common.annotations.VisibleForTesting;
-
+import sun.misc.Unsafe;
```

Comment:
Can not do this change and below in this file.  We dont want to directly refer Unsafe class.  This can cause ClassNotFoundError for some JVMs.  That is why we hide the usage under UnsafeAccess and there via reflection we load and catch Exception also.

Reply:
ok ,  i will remove Unsafe import , and i notice this too but don't know why ,thanks for point it